eks-deployments/patch-coredns/README.md

```diff
@@ -22,6 +22,8 @@ data:
         health
         kubernetes cluster.local in-addr.arpa ip6.arpa {
         pods insecure
+        # Note that the keyword "upstream" is deprecated starting from Kubernetes 1.18,
```

Comment:
I think placing the comment here i.e. in between the expected output can be confusing for the customers. 
Put two expected outputs -- one for v1.16 & 1.17 and another one for v1.18 and v1.19 and before the output comment about the upstream keyword.